zebra-rpc/src/methods/tests/snapshot/get_block_template_rpcs.rs

```diff
@@ -75,17 +81,28 @@ pub async fn test_responses<State, ReadState>(
         miner_address: Some(transparent::Address::from_script_hash(network, [0xad; 20])),
     };
 
+    // nu5 block height
+    let fake_tip_height = NetworkUpgrade::Nu5.activation_height(network).unwrap();
+    // nu5 block hash
+    let fake_tip_hash =
+        Hash::from_hex("0000000000d723156d9b65ffcf4984da7a19675ed7e2f06d9e5d5188af087bf8").unwrap();
+    // nu5 block time
+    let fake_tip_time = Utc.timestamp_opt(1654008605, 0).unwrap();
```

Comment:
Can we make the test data obey the consensus rule that the `cur_time` must be equal or greater than the `min_time`?

It doesn't really matter for this test, but it makes reading the snapshot a bit confusing.

```rust
    // nu5 block time + 1
    let fake_tip_time = Utc.timestamp_opt(1654008606, 0).unwrap();
```

Reply:
Fixed by eef133c